### DISCUSSION TITLE
Converting eps to svg error in illustrator

### DISCUSSION THREAD

**Discussion by oleksiivasylenko** [dlemstra/Magick.NET#610] [General] [answered]
I have very small app in c# which should convert .eps to .svg. Using Magic.Net-Q16-AnyCPU as a nuget also installed on windows GhostScript gs952w32.exe.

Function for conversion:

```
using (MagickImage image = new MagickImage(@"C:\folder1\1.eps"))
{
    image.Format = MagickFormat.Svg;
    image.Write(@"C:\folder2\1.svg");
}
```
image is converted and also I can open it via some browsers websites which can read svg. But when I try it open with illustrator it shows an error "Could not find the linked file ''. Choose replace to select another file or Ignore to leave the link unchanged." And after pressing ignore an image is empty.

**Comment by dlemstra** [comment]
Can you share `1.eps` ?

**Comment by oleksiivasylenko** [comment]
Hi @dlemstra. Yes here it is, with video example
[Example.zip](https://github.com/dlemstra/Magick.NET/files/4480895/Example.zip)

**Comment by dlemstra** [comment, marked as answer]
Found a post from 2018 where it seems that Illustrator does not support embeded PNG files: https://community.adobe.com/t5/illustrator/illustrator-does-not-support-my-svg-file-that-contains-svg-image-elements/td-p/9651571?page=1.

And ImageMagick/Magick.NET is probably also not the best tool to convert from EPS to SVG. The image will be changed from a vector format to a raster image. Don't know what other tool I could advise you though.

**Reply by oleksiivasylenko** [reply]
Yes, probably it related to illustrator.

You are right about tool for these types conversion, perhaps there is no good instrument for this... 
The issue can be closed in such case. Thanks.
